.all-contributorsrc

```diff
@@ -1198,6 +1198,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "twhite96",
```

Comment:
Isn't this related to another commit?

Reply:
I think this is because of merging `master` back into this branch